<launch>
  <!-- Sample launch file for xtion_grabber. This assumes you installed the
       udev rules as detailed in README.md. -->

  <!-- The ID of your xtion if a specific one should be used. You can look this up with dmesg. -->
  <arg name="ID" default="1306060188" />
  <arg name="name" default="camera" />
  <arg name="tf_prefix" default="" />
  <node name="$(arg name)" pkg="nodelet" type="nodelet" args="standalone xtion_grabber" output="screen">
    <param name="color_device" value="/dev/xtion_$(arg ID)_color" />
    <param name="depth_device" value="/dev/xtion_$(arg ID)_depth" />

    <param name="color_width" value="640" />
    <param name="color_height" value="480" />
    
    <param name="tf_prefix" value="$(arg tf_prefix)" />
  </node>
</launch>
